src/Graph5/Graph5.tsx

```diff
@@ -0,0 +1,46 @@
+import { AreaChart, Area, ResponsiveContainer } from "recharts";
+import data from "./DataGraph5.json";
+
+function findHighestUV(data: { name: string; uv: number }[]): number {
+	const uvValues = data.map((item) => item.uv);
+	const highestUV = Math.max(...uvValues);
+	return highestUV;
+}
+
+const highestUVValue = findHighestUV(data);
```

Comment:
Nazwij prosze zmienna duzymi literami, to taka dobra praktyka w reakcie na nazywanie zmiennych ktore beda "stale", bo ta wartosc raz wyliczona juz nigdy sie nie zmieni

Reply:
done